src/components/Animals/Search.js

```diff
@@ -0,0 +1,34 @@
+// import React, { useEffect, useState } from 'react';
+// import { useSelector } from 'react-redux';
+
+// const Search = () => {
+//   const animals = useSelector((state) => state.animals);
+//   const [newAnimals, setNewAnimals] = useState([]);
+
+//   useEffect(() => {
+//     setNewAnimals(animals);
+//   }, [animals, setNewAnimals]);
+
+//   const handleSearch = (e) => {
+//     e.preventDefault();
+//     let myValue = e.target.value;
+//     if (myValue.length > 0) {
+//       myValue = myValue.toLowerCase();
+//       const filteredAnmls = animals.filter((animal) =>
+// animal.name.toLowerCase().includes(myValue));
+//       setNewAnimals(filteredAnmls);
+//     } else {
+//       setNewAnimals(animals);
+//     }
+//   };
+
+//   return (
+//     <div>
+//       <div className="searchbar">
+//         <input type="text" placeholder="Search eg Lion..." onChange={handleSearch} />
+//       </div>
+//     </div>
+//   );
+// };
+
+// export default Search;
```

Comment:
- [ ] Kindly remove this file of commented code if you would not be using it, but get rid of the commented code. It makes your code look messy

Reply:
> * [ ]  Kindly remove this file of commented code if you would not be using it, but get rid of the commented code. It makes your code look messy

- [ ] As requested by the previous reviewer, please remove do not leave unused code as a comment in your codebase. this will make your code much more noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻